Library/Homebrew/shims/mac/super/make

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
-if [[ -n "$HOMEBREW_MAKE" && "$HOMEBREW_MAKE" != "make" ]]
+# shellcheck disable=SC2154
```

Comment:
I think the use of `HOMEBREW_MAKE` can be removed as it's undocumented and unset.

Reply:
>  the use of HOMEBREW_MAKE can be removed

I understood, I will open another PR to remove HOMEBREW_MAKE statement from `shims/mac/super/make`, `shims/linux/super/make` and `shims/mac/super/gmake`